packages/editor/src/components/block-inspector/index.js

```diff
@@ -28,30 +30,44 @@ const BlockInspector = ( { selectedBlock, blockType, count } ) => {
 		return <span className="editor-block-inspector__no-blocks">{ __( 'No block selected.' ) }</span>;
 	}
 
-	return [
-		<div className="editor-block-inspector__card" key="card">
-			<BlockIcon icon={ blockType.icon } showColors />
-			<div className="editor-block-inspector__card-content">
-				<div className="editor-block-inspector__card-title">{ blockType.title }</div>
-				<div className="editor-block-inspector__card-description">{ blockType.description }</div>
+	return (
+		<Fragment>
+			<div className="editor-block-inspector__card">
+				<BlockIcon icon={ blockType.icon } showColors />
+				<div className="editor-block-inspector__card-content">
+					<div className="editor-block-inspector__card-title">{ blockType.title }</div>
+					<div className="editor-block-inspector__card-description">{ blockType.description }</div>
+				</div>
 			</div>
-		</div>,
-		<div key="inspector-controls"><InspectorControls.Slot /></div>,
-		<div key="inspector-advanced-controls">
-			<InspectorAdvancedControls.Slot>
-				{ ( fills ) => ! isEmpty( fills ) && (
+			<div><InspectorControls.Slot /></div>
+			{ !! blockType.styles && (
+				<div className="editor-block-inspector__styles">
 					<PanelBody
-						className="editor-block-inspector__advanced"
-						title={ __( 'Advanced' ) }
+						title={ __( 'Styles' ) }
 						initialOpen={ false }
 					>
-						{ fills }
+						<BlockStyles
+							clientId={ selectedBlock.clientId }
+						/>
 					</PanelBody>
-				) }
-			</InspectorAdvancedControls.Slot>
-		</div>,
-		<SkipToSelectedBlock key="back" />,
-	];
+				</div>
+			) }
+			<div>
```

Comment:
Why is this div needed?

Reply:
Yeah, this should be improved at some point. We had to keep it when we refactored Slots to stop using wrapping `div`, some styles depend on that fact. We should clean it up sooner than later.

Reply:
Yes, the styling of the panels in the block inspector is a bit weird (they use negative margins) which makes a wrapper necessary. I think we should rethink the styles of the `Panel`/ `PanelBody` and `PanelHeader` entirely.

Reply:
Ok, I trust you all.